content/dstu2/Authorization.md

```diff
@@ -0,0 +1,156 @@
+# Authorization Server Client Documentation #
+---------------------------------------------
+
+### Introduction ###
+The Cerner Authorization Server currently supports OAuth 2.0 [1] SMART [2, 3] on FHIR launch workflows. As a client, it will require interaction between the client (your client application), the user, the authorization server, a SMART Launch server and a FHIR resource server.
```

Comment:
References to "SMART Launch server" are likely to confuse developers since this is an internal implementation detail in our system. Others implementing SMART may bake this directly into the EHR app rather than the approach we've taken. I suggest we just omit all references to this and instead use 'EHR' which is what the SMART documentation uses.